Cargo.toml

```diff
@@ -80,7 +81,7 @@ lto           = "fat"
 codegen-units = 1
 strip         = "symbols"
 debug         = false
-panic         = "abort"   # Let it crash and force ourselves to write safe Rust.
+panic         = "abort" # Let it crash and force ourselves to write safe Rust.
```

Comment:
Can you move this to CLI only? There are other binaries that require cleanups on `drop`.

Reply:
Also, how would setting `panic` to `abort` make `pacquet` faster (on happy path)?

Reply:
Ah, sorry. This change is mostly my mistake while profiling.